jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseServiceImplTest.java

```diff
@@ -1840,20 +1840,27 @@ public void testCaseRolesWithQueries() {
             instances = caseRuntimeDataService.getCaseInstancesAnyRole(status, new QueryContext());
             assertNotNull(instances);
             assertFalse("Opened case was returned when searching for cancelled case instances.", instances.stream().anyMatch(n -> n.getCaseId().equals(caseId)));
-
+            
             status = Arrays.asList(CaseStatus.OPEN);
+            
+            instances = caseRuntimeDataService.getCaseInstancesOwnedBy("john", status, true, new QueryContext());
+            assertNotNull(instances);
+            assertEquals(1, instances.size());
+            assertNotNull(instances.iterator().next().getCaseFile());
```

Comment:
As in the previous PR, I would specifically create a test for that to not mix the focus of the test. For example this one is called `testCaseRolesWithQueries` so I don't expect it to test CaseFile retrieval for case instances.